lib/confluence/api_velocity.es6.js

```diff
@@ -25,14 +25,14 @@ foam.CLASS({
     {
       name: 'browserApiDAO',
       documentation: `A DAO containing junction objects of Browser and
-        WebInterface.`,
+          WebInterface.`,
       typeName: 'BrowserAPI DAO',
       required: true,
       final: true,
     },
     {
       name: 'browserDAO',
-      documentation: `A DAO containing all browser infomation.`,
+      documentation: `A DAO containing all browser information.`,
```

Comment:
nit: "all [...] information" is a bit vague. Maybe `each browsers' version and environment metadata` or similar?